cli/cmdlineparser.cpp

```diff
@@ -1191,6 +1198,7 @@ void CmdLineParser::printHelp() const
         "    -f, --force          Force checking of all configurations in files. If used\n"
         "                         together with '--max-configs=', the last option is the\n"
         "                         one that is effective.\n"
+        "    --funsigned-char     Treat char type as unsigned.\n"
```

Comment:
if we list the options alphabetically it seems to me that --fsigned-char and --funsigned-char should be listed together.

Reply:
Yes, `--fsigned-char` is placed incorrectly. Total mindblank on that addition. :|

Reply:
Done.